### PR TITLE
Create LukeMurray.json

### PR DESCRIPTION
**Pull request by lukemurraynz**
## Changes proposed

Added: New JSON file for: Luke Murray content for LinkFree profile.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/931"></a>

### CODE REVIEW

File: public/data/LukeMurray.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Luke Murray",
+    "type": "personal",
+    "bio": "Technical Consultant, Azure Solutions Architect Expert, Technologist and a drinker of coffee.",
+    "avatar": "https://luke.geek.nz/images/ProfileRealLife.jpg",
+    "links": [
+      {
+        "name": "Follow me on my Website",
+        "url": "https://luke.geek.nz",
+        "icon": "Globe"
```
Comment (syedsohan) [inline public/data/LukeMurray.json:10]:
```suggestion
        "icon": "globe"
```

File: public/data/LukeMurray.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/ljmurray",
+        "icon": "linkedin"
+      },
+      {
+        "name": "Follow me on Microsoft Tech Community",
+        "url": "https://techcommunity.microsoft.com/t5/user/viewprofilepage/user-id/108279#profile",
+        "icon": "Microsoft"
```
Comment (syedsohan) [inline public/data/LukeMurray.json:30]:
```suggestion
        "icon": "microsoft"
```

File: public/data/LukeMurray.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        "url": "https://luke.geek.nz",
+        "icon": "Globe"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/lukemurraynz",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on Github",
```
Comment (syedsohan) [inline public/data/LukeMurray.json:18]:
```suggestion
        "name": "Follow me on GitHub",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

**Review by syedsohan — APPROVED** [review]

**Review by emmalearnscode — APPROVED** [review]

**Review by krishguptadev — APPROVED** [review]

File: public/data/LukeMurray.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        "icon": "linkedin"
+      },
+      {
+        "name": "Follow me on Microsoft Tech Community",
+        "url": "https://techcommunity.microsoft.com/t5/user/viewprofilepage/user-id/108279#profile",
+        "icon": "microsoft"
+      }   
+    
+    ]
+  }
```
Comment (Pradumnasaraf) [inline public/data/LukeMurray.json:34]:
```suggestion
{
  "name": "Luke Murray",
  "type": "personal",
  "bio": "Technical Consultant, Azure Solutions Architect Expert, Technologist and a drinker of coffee.",
  "avatar": "https://luke.geek.nz/images/ProfileRealLife.jpg",
  "links": [
    {
      "name": "Follow me on my Website",
      "url": "https://luke.geek.nz",
      "icon": "globe"
    },
    {
      "name": "Follow me on Twitter",
      "url": "https://twitter.com/lukemurraynz",
      "icon": "twitter"
    },
    {
      "name": "Follow me on GitHub",
      "url": "https://github.com/lukemurraynz",
      "icon": "github"
    },
    {
      "name": "Follow me on LinkedIn",
      "url": "https://www.linkedin.com/in/ljmurray",
      "icon": "linkedin"
    },
    {
      "name": "Follow me on Microsoft Tech Community",
      "url": "https://techcommunity.microsoft.com/t5/user/viewprofilepage/user-id/108279#profile",
      "icon": "microsoft"
    }     
  ]
}
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
#### Changes
- Indentation issue
- Extra line no 32

File: public/data/LukeMurray.json (hunk 4 above)
Comment (krishguptadev) [reply public/data/LukeMurray.json:34]: Please do not make suggestions related to formatting since it is now done by the GH action 🙂

Comment (Pradumnasaraf) [reply public/data/LukeMurray.json:34]: Ohh, great 🙌

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**